gradle/compile-js-multiplatform.gradle

```diff
@@ -6,12 +6,20 @@ apply from: rootProject.file('gradle/node-js.gradle')
 
 kotlin {
     targets {
-        fromPreset(presets.js, 'js')
+        fromPreset(presets.js, 'js') {
+            // Enable built-in test runner only for IR target.
+            // These runners don't support changing js module name change.
+            if (js.hasProperty("irTarget")) {
+                irTarget.browser()
+                irTarget.nodejs()
+            }
+        }
     }
 
     sourceSets {
         jsMain.dependencies {
             api "org.jetbrains.kotlin:kotlin-stdlib-js:$kotlin_version"
+            api "org.jetbrains.kotlinx:atomicfu-js:$atomicfu_version"
```

Comment:
dependency should be conditional and present only for IR backend

Reply:
It might be hard to enable it conditionally in this build script. @ilgonmic can you help?

Reply:
It seems that it should be not dependency in js source set but specific ir target compilation
```
js().irTarget?.compilations['main']?.dependencies {
   api "org.jetbrains.kotlinx:atomicfu-js:$atomicfu_version"
}

```